src/bson.rs

```diff
@@ -1054,6 +1054,99 @@ impl Binary {
     }
 }
 
+#[cfg(feature = "uuid-0_8")]
+#[cfg_attr(docsrs, doc(cfg(feature = "uuid-0_8")))]
+#[derive(PartialEq)]
+pub enum UuidRepresentation {
```

Comment:
just to be safe, I think we'll need to tag this with `#[non_exhaustive]`. That way it won't be considered a breaking change to add a new case in the future.